package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "hapi-graphql-2",
   "version": "2.0.0",
-  "description": "Create a GraphQL HTTP server with Hapi.",
-  "author": "Simon Degraeve <simon.degraeve@gmail.com>",
+  "description": "Create a GraphQL HTTP server with Hapi v17",
```

Comment:
I am guessing we will make the package work with Hapi v18 when the time comes?

Reply:
so maybe it makes sense to stick with "...server with Hapi"

Reply:
fixed